api/src/functionalTest/java/uk/gov/hmcts/reform/divorce/caseapi/SaveAndSignOutNotificationTest.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.hmcts.reform.divorce.caseapi;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.divorce.caseapi.model.ccd.CcdCallbackRequest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.reform.divorce.ccd.model.enums.DivorceOrDissolution.DIVORCE;
+
+@SpringBootTest
+public class SaveAndSignOutNotificationTest {
+    public static final String TEST_USER_EMAIL = "simulate-delivered@notifications.service.gov.uk";
+    public static final String TEST_FIRST_NAME = "John";
+    public static final String TEST_LAST_NAME = "Smith";
+
+    public static final String D8_PETITIONER_FIRST_NAME = "D8PetitionerFirstName";
+    public static final String D8_PETITIONER_LAST_NAME = "D8PetitionerLastName";
+    public static final String D8_PETITIONER_EMAIL = "D8PetitionerEmail";
+    public static final String DIVORCE_OR_DISSOLUTION = "divorceOrDissolution";
+    public static final String NOTIFY_APPLICANT = "/notify-applicant";
+
+    @Value("${test-url}")
```

Comment:
I don't think this should be configurable

Reply:
as discussed on slack it is only host which makes it easier for local testing